stop-es-cluster.sh

```diff
@@ -0,0 +1,12 @@
+kubectl config use-context elk
```

Comment:
Yes. I thought it is a best practice to separate out the context of this environment from default or other Kubernetes context.